qnm/angular.py

```diff
@@ -81,7 +81,30 @@ def swsphericalh_A(s, l, m):
     return l*(l+1) - s*(s+1)
 
 def M_matrix_elem(s, c, m, l, lprime):
-    """ Eq. (55) """
+    """ Matrix elements from Eq. (55). 
```

Comment:
This is my fault, since I was very lazy documenting the first few functions I wrote. Just giving an Eq. here is way too vague. This should be something like "The (l, lprime) matrix element of the spherical-spheroidal decomposition matrix.